modules/Documentation/components/ApiEndpoints/index.tsx

```diff
@@ -3,27 +3,32 @@ import Text from '../../../../components/Text';
 import Button from '../../../../components/Button';
 import { Code } from '../../index.styled';
 import Snippet from '../Snippet';
+import { FormattedMessage } from 'react-intl'
+import messages from './index.messages'
 
 export default ({ GRAPHQL_SERVER_URI, clientId, clientSecret }) => (
   <>
     <Text mt="8px">
-      Endpoints provide interface for retrieving information about payments on
-          your back-end server. Vanilla currently provides{' '}
+      <FormattedMessage {...messages.intro} />
+      {' '}
       <Button
         secondary
         medium
         as="a"
         href={`https://${GRAPHQL_SERVER_URI}`}
         target="_blank"
       >
-        GraphQL API.
-      </Button>
+        GraphQL API
+      </Button>.
     </Text>
     <Text mt="24px" semibold>
-      Authentication
-        </Text>
+      <FormattedMessage {...messages.authenticationTitle} />
+    </Text>
     <Text mt="4px">
-      For requests authentication use generated <Code>clientID</Code> and{' '}
+      <FormattedMessage {...messages.whatToUse.first} />
+      <Code>clientID</Code>
+      <FormattedMessage {...messages.whatToUse.second} />
+      {' '}
       <Code>clientSecret</Code>.
```

Comment:
U dont have to split the text to 2 parts u can use React component

Example:
**Sentence**
`For requests authentication use generated <Code>clientID</Code> and  <Code>clientSecret</Code>. `
**Message**
`<FormattedMessage {...messages.whatToUse}  code={(value)=><Code>{value}</Code>} />`

**Defined message**
```
 whatInclude: {
      id: 'Authentication.whatInclude',
      defaultMessage: 'For requests authentication use generated <code>clientID</code> and <code>clientSecret</code>.'
    }
  },

```